### PR TITLE
Update containerd-env-setup.md

### PR DESCRIPTION
**Pull request by aa624545345**
Add the description of the two nydusd binaries.

### CODE REVIEW

**Comment by imeoer** [comment]
@aa624545345 Thanks for the contribution, please add SOB (signed off by) for the commit, thanks! :)

File: docs/containerd-env-setup.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ This document will walk through how to setup a nydus image service to work with
 
 Get `nydus-image`, `nydusd`, `nydusify`, `ctr-remote` and `nydus-overlayfs` binaries from [image-service release](https://github.com/dragonflyoss/image-service/releases/latest) page.
 
+`nydusd` currently has two binaries, `nydusd-fusedev` (for fuse daemon mode) and `nydusd-virtiofs` (for virtiofs daemon mode), chose the one that fits your mode and rename it `nydusd`.
```
Comment (changweige) [inline docs/containerd-env-setup.md:9]: We have a typo here? Should it the correct one be "choose"?
And "rename it as `nydusd` "would be better?

**Comment by yqleng1987** [comment]
@aa624545345 , your pull request has been updated. A new test job will be submitted. Please wait in patience.

**Comment by yqleng1987** [comment]
@aa624545345 , The CI test is completed, please check result:
<table>
    <tr>
        <th colspan="2">Test Case</th><th>Test Result</th>
    </tr>
    <tr>
        <td colspan="2">merge-target-branch</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5443>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">build-docker-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5443>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5443>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-ctr-remote</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5443>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus-snapshotter</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5443>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">start-nydus-snapshotter-config-containerd</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5443>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">run-container-with-nydus-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5443>:white_check_mark:SUCCESS</a></td>
    </tr>
</table>

Congratulations, your test job passed!

**Comment by yqleng1987** [comment]
@aa624545345 , your pull request has been updated. A new test job will be submitted. Please wait in patience.

**Comment by yqleng1987** [comment]
@aa624545345 , your test job has passed, and no need to test again.

**Comment by changweige** [comment]
We still can't merge this PR since it fails DCO check.
Please try to refer to https://github.com/dragonflyoss/image-service/pull/505/checks?check_run_id=6986289482 for further information and then tweak your patch commit message.

**Comment by yqleng1987** [comment]
@aa624545345 , your pull request has been updated. A new test job will be submitted. Please wait in patience.

**Review by changweige — APPROVED** [review]

**Comment by yqleng1987** [comment]
@aa624545345 , The CI test is completed, please check result:
<table>
    <tr>
        <th colspan="2">Test Case</th><th>Test Result</th>
    </tr>
    <tr>
        <td colspan="2">merge-target-branch</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5532>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">build-docker-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5532>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5532>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-ctr-remote</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5532>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">compile-nydus-snapshotter</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5532>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">start-nydus-snapshotter-config-containerd</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5532>:white_check_mark:SUCCESS</a></td>
    </tr>
    <tr>
        <td colspan="2">run-container-with-nydus-image</td><td><a href=https://tone.openanolis.cn/ws/nrh4nnio/test_result/5532>:white_check_mark:SUCCESS</a></td>
    </tr>
</table>

Sorry, the current test job result is not valid, because your pull request has been updated, and a new test job will be submitted.

**Pull request merged**